test-cases/performance/perf-regression-throughput-650gb.yaml

```diff
@@ -0,0 +1,75 @@
+  "cql-stress-cassandra-stress write no-warmup cl=ALL n=108333334 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -col 'size=FIXED(128) n=FIXED(8)'  -pop seq=433333337..541666670",
+  "cql-stress-cassandra-stress write no-warmup cl=ALL n=108333334 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -col 'size=FIXED(128) n=FIXED(8)'  -pop seq=541666671..650000004"
+]
+
+stress_cmd_w: [
+  "cql-stress-cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=220 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..325000000",
+  "cql-stress-cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=220 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=325000001..650000000",
+  "cql-stress-cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=220 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=650000001..975000000",
+  "cql-stress-cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=220 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=975000001..1300000000"
+]
+stress_cmd_r: [
+  "cql-stress-cassandra-stress read no-warmup cl=QUORUM duration=60m -mode cql3 native -rate threads=380 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..108333334,54166668,542209)'",
+  "cql-stress-cassandra-stress read no-warmup cl=QUORUM duration=60m -mode cql3 native -rate threads=380 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(108333335..216666668,162500002,542209)'",
+  "cql-stress-cassandra-stress read no-warmup cl=QUORUM duration=60m -mode cql3 native -rate threads=380 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(216666669..325000002,270833336,542209)'",
+  "cql-stress-cassandra-stress read no-warmup cl=QUORUM duration=60m -mode cql3 native -rate threads=380 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(325000003..433333336,379166670,542209)'",
+  "cql-stress-cassandra-stress read no-warmup cl=QUORUM duration=60m -mode cql3 native -rate threads=380 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(433333337..541666670,487500004,542209)'",
+  "cql-stress-cassandra-stress read no-warmup cl=QUORUM duration=60m -mode cql3 native -rate threads=380 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(541666671..650000004,595833338,542209)'"
+]
+stress_cmd_m: [
+  "cql-stress-cassandra-stress mixed no-warmup cl=QUORUM duration=60m -mode cql3 native -rate threads=350 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..108333334,54166668,542209)'",
```

Comment:
I don't know if we have version that supports mixed in SCT yet.

I suggest testing it locally before trying it in a job, like all of those command, we might found it doesn't exactly support all the arguments we are passing

Reply:
I try first write and later manually try to find params for read and see if mixed is supported